decidim-participatory_processes/spec/system/participatory_process_groups_spec.rb

```diff
@@ -454,129 +440,25 @@
         visit decidim_participatory_processes.participatory_process_group_path(participatory_process_group)
       end
 
-      it "lists active processes ordered by start_date" do
-        within "#processes-grid" do
-          expect(titles.count).to eq(3)
-          expect(titles[0].text).to eq(translated(active_process_with_area.title, locale: :en))
-          expect(titles[1].text).to eq(translated(active_process_with_scope.title, locale: :en))
-          expect(titles[2].text).to eq(translated(active_process.title, locale: :en))
+      it "lists active processes ordered by weigtht" do
+        within "section.content-block" do
+          expect(titles[0].text).to eq(translated(upcoming_process_with_area.title, locale: :en))
+          expect(titles[1].text).to eq(translated(active_process_with_area.title, locale: :en))
+          expect(titles[2].text).to eq(translated(past_process_with_scope.title, locale: :en))
+          expect(titles[3].text).to eq(translated(active_process.title, locale: :en))
+          expect(titles[4].text).to eq(translated(active_process_with_scope.title, locale: :en))
         end
       end
 
-      it_behaves_like "showing all processes counts"
```

Comment:
How are these tests been replaced?

Reply:
This test currently has been removed, but once moved the processes count metadata to the main data content block it could be recovered and checked there

Reply:
I've added a similar test to find the count when the title content block is active but the "showing all processes count" shared examples are related with the horizontal filter which is currently deprecated